.github/workflows/pr-smoke-test.yml

```diff
@@ -49,8 +51,50 @@ jobs:
           DEFAULT_TIMEOUT_OVERRIDE: 120000
         run: |
           pnpx wc-e2e test:e2e plugins/woocommerce/tests/e2e/specs/smoke-tests/update-woocommerce.js
+
+      - name: Post Smoke tests results comment on PR
+        if: always()
+        uses: actions/github-script@v5
+        env: 
+          TITLE: 'Smoke Test Results:'
+          SMOKE_TEST_URL: ${{ secrets.SMOKE_TEST_URL }}
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const script = require( '@woocommerce/packages/js/e2e-environment/bin/post-results-to-github-pr' )
```

Comment:
This still has the `packages/js/` in the package name/path so CI can''t find it. In this repo `@woocommerce/e2e-environment` maps to `packages/js/e2e-environment`.

Reply:
Thanks for catching that 😅. The paths are now fixed.